Include/DSP_constants.h

```diff
@@ -6,28 +6,21 @@
 #ifndef ACOUSTICS_DSP_CONSTANTS_H
 #define ACOUSTICS_DSP_CONSTANTS_H
 
-/**
- * @brief Important defines that unlocks multiple of the functions
- * and defines from the include-files
- */
-#ifndef STM32_DEFINES
-#define STM32_DEFINES
-  
-  #define STM32F767xx               /* Current MCU used for the project           */
-  #define ARM_MATH_CM7              /* Processor-version used on the STM32        */
-  #define _USE_MATH_DEFINES         /* Gives access to math-defines such as M_PI  */
-  #define __SOFTFP__                /* Allows assert by software                  */
-  #define HAL_ADC_MODULE_ENABLED    /* Enables functions operating on hardware    */
-
-#endif /* STM32_DEFINES */
-
 #include <stdint.h>
 #include <math.h>
 #include <cmath>
 #include <utility>
 #include "arm_math.h"
 #include "arm_const_structs.h" 
 
+/**
+ * @brief Duplicating code is terrible code-practice! These constants somehow 
+ * throws an error during compilation. Therefore tried to move the variables 
+ * to eliminate such error 
+ */
+float32_t pi   = 3.14159265358979323846;        /* PI (obviously)               */
```

Comment:
👀

Reply:
Heh. Changed to Pi ;)

Don't understand why the compiler/linker throws an error from _MATH_H::M_PI 
Getting it to work would be preferable